packages/react/src/components/form/initial/index.tsx

```diff
@@ -78,5 +117,6 @@ Initial.Logo = LogoSlot;
 Initial.Header = HeaderSlot;
 Initial.Controls = Controls;
 Initial.OIDC = OIDCSlot;
```

Comment:
Can we try marking this export deprecated? Some tips on how to do it [here](https://github.com/palantir/tslint/issues/3751#issuecomment-370265475)